pkg/subvolume/subvolume.go

```diff
@@ -0,0 +1,145 @@
+/*
```

Comment:
Let's put this under the folder `pkg/filesystem/subvolume.go`. Then other commands related to filesystem in the future can be added to this package.